atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -339,7 +339,7 @@ private boolean loadKeyStores() {
         logKeyStore(ks, ksLocation, ksPwd);
       }
     } catch (final FileNotFoundException e) {
-      throw new AtomixRuntimeException("Could not load cluster keystore: {}", e.getMessage());
+      throw new AtomixRuntimeException("Could not load cluster keystore: %s", e.getMessage());
```

Comment:
Can't tell if it's clever to accept a format or confusing :sweat_smile:

Reply:
Yeah didn't wanted to rewrite the complete stuff :man_shrugging: